utils/__tests__/config.test.ts

```diff
@@ -1,8 +1,10 @@
 import * as E from "fp-ts/lib/Either";
 
-import { IConfig } from "../config";
+import { IConfig, UlidMapFromString } from "../config";
 
 import { envConfig } from "../../__mocks__/env-config.mock";
+import { JsonFromString } from "io-ts-types";
+import { Ulid } from "@pagopa/ts-commons/lib/strings";
```

Comment:
`JsonFromString` and `Ulid` are unused, also it seems that in the `package.json` we added `io-ts-types` as a dependency to import `JsonFromString` in this file, so i think we can remove this dependency as well

Reply:
yes we can remove this and the io-ts-types and monocle-ts and newtype-ts that were needed for io-ts-type